velocidi-sdk/src/test/kotlin/com/velocidi/VelocidiTest.kt

```diff
@@ -41,15 +42,37 @@ class VelocidiTest {
 
         val context: Context = ApplicationProvider.getApplicationContext()
 
+        val appView = """
```

Comment:
Can you add a second event example, where we try to use arrays?

Reply:
Changed `customFields.roles` to be an array in ef64838